frontend/packages/dev-console/src/components/import/advanced/ResourceLimitSection.tsx

```diff
@@ -1,12 +1,15 @@
 import * as React from 'react';
 import { ResourceLimitField } from '@console/shared';
-import { useField } from 'formik';
+import { useFormikContext, FormikValues } from 'formik';
 import FormSection from '../section/FormSection';
 import { MemoryUnits, CPUUnits } from '../import-types';
 
 const ResourceLimitSection: React.FC = () => {
-  const [cpuLimits] = useField('limits.cpu');
-  const [memoryLimits] = useField('limits.memory');
+  const {
+    values: {
+      limits: { cpu, memory },
+    },
+  } = useFormikContext<FormikValues>();
```

Comment:
@divyanshiGupta why is it we need to use `useFormikContext` here but we still use `useField` elsewhere like here:
https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/import/git/SourceSecretSelector.tsx#L12

Reply:
@christianvogt after upgrade it was throwing error on line 18,27,37 and 46 that the `defaultRequestUnit` does not exist on type string, I tried to do a bunch of things but the error wont go, then I tried to use the same values from `useFormikContext` and the error was gone.